js/packages/eyes-storybook/src/generateConfig.js

```diff
@@ -75,7 +76,24 @@ function generateConfig({argv = {}, defaultConfig = {}, externalConfigParams = [
       result.testConcurrency,
     );
   }
+
+  if (!result.browser) {
+    result.renderers = [{name: 'chrome', width: 1024, height: 768}];
+  } else {
+    result.renderers = [];
+    if (!Array.isArray(result.browser)) {
+      result.browser = [result.browser];
+    }
+    for (const browser of result.browser) {
+      if (browser.deviceName) {
+        result.renderers.push({
+          chromeEmulationInfo: browser,
+        });
+      } else {
+        result.renderers.push(browser);
+      }
+    }
```

Comment:
What about doing it like this? A little bit more compact
```suggestion
    result.renderers = result.browser.map(browser => {
      return browser.deviceName ? {chromeEmulationInfo: browser} : browser
    })
```